core/domain/topic_services.py

```diff
@@ -261,7 +263,7 @@ def apply_change_list(topic_id, change_list):
     try:
         for change in change_list:
             if change.cmd == topic_domain.CMD_ADD_SUBTOPIC:
-                topic.add_subtopic(change.id, change.title)
+                change.subtopic_id = topic.add_subtopic(change.title)
```

Comment:
It seems a bit weird to modify the change object like this, while you're supposed to be applying changes _from_ the change object. Is there any precedent?

Reply:
Done, removed this.